pkg/config/settings/http/server.go

```diff
@@ -129,7 +131,15 @@ func getConfigValue(w http.ResponseWriter, r *http.Request) {
 		}
 		return
 	}
-	body, err := json.Marshal(map[string]interface{}{"value": val})
+
+	resp := map[string]interface{}{"value": val}
+	if r.URL.Query().Get("sources") == "true" {
+		sourcesHierarchy, sourcesVal := config.Datadog.GetAllSources(setting)
+		resp["sources_value"] = sourcesVal
+		resp["sources_hierarchy"] = sourcesHierarchy
+	}
```

Comment:
If we care about the order we need to return a list of pairs rather than 2 objects (ie: having to correlate 2 values is error prone).

Reply:
Yess that's perfect!